src/ClassRegistrator_impl.hpp

```diff
@@ -0,0 +1,139 @@
+#include <string>
+
+#include "sexp.hpp"
+
+namespace { // Implementation detail, so it's in an anon namespace
```

Comment:
I don't think we should indent namespace contents. Usually namespace spans across all file, so adding indent level is too much, and doesn't convey any useful information.

Reply:
Well, this one does not span the whole file. It's a private implementation thing.

Reply:
Doesn't matter, there's at max three namespaces per file, and when most of the code lies in a namespace, it doesn't make sense to indent it, since it doesn't convey any useful information.

Reply:
Makes sense, i guess.